content/board/FlorianHeigwer.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Florian Heigwer"
+tagline: "Event officer"
+date: 2018-07-13
+publishdate: 1970-01-01
+weight: 10
+image: boardAvatars/FlorianHeigwer.jpg
+---
+
+
```

Comment:
plsease add:

I am a senior PostDoc at the [German Cancer Research Center](https://www.dkfz.de/en/index.html) working in functional genomics, high throughput screening and bioinformatics.

I have a background in molecular biotechnology and bioinformatics.